ref/net46/Microsoft.Build/Microsoft.Build.cs

```diff
@@ -1402,6 +1425,25 @@ public partial class DistributedFileLogger : Microsoft.Build.Framework.IForwardi
         public void Initialize(Microsoft.Build.Framework.IEventSource eventSource, int nodeCount) { }
         public void Shutdown() { }
     }
+    public partial class EventArgsDispatcher : Microsoft.Build.Framework.IEventSource
```

Comment:
Is there a need to have these public? Seems like something to avoid when possible. Even the `BinaryLogger` class, although the `FileLogger` is public so maybe there's a reason?